plugins/inputs/mqtt_consumer/mqtt_consumer.go

```diff
@@ -96,10 +96,6 @@ var sampleConfig = `
     "telegraf/+/mem",
     "sensors/#",
   ]
-  ## Enable extracting tag values from MQTT topics
-  ## _ denotes an ignored entry in the topic path
-  # topic_tags = "_/format/client/_"
-  # topic_measurement = "measurement/_/_/_"
```

Comment:
Did you mean to delete this?

Reply:
Yes as it is further down in the readme and should not be in both places

Reply:
ah, good catch!